app/views/users/_form.html.erb

```diff
@@ -1,32 +1,24 @@
 <%= form_with(model: user, url: signup_path, class:"mx-1 mx-md-4")  do |form| %>
```

Comment:
This form should not be submitted as a signup. It should be users create and get redirected to users index after creation.